ui/src/components/AwsProvisionerWorkerTypeStatus/index.jsx

```diff
@@ -112,19 +112,19 @@ export default class AwsProvisionerWorkerTypeStatus extends Component {
     }
   );
 
-  handleHeaderClick = sortBy => {
+  handleHeaderClick = header => {
     const toggled = this.state.sortDirection === 'desc' ? 'asc' : 'desc';
-    const sortDirection = this.state.sortBy === sortBy ? toggled : 'desc';
+    const sortDirection = this.state.sortBy === header.id ? toggled : 'desc';
 
-    this.setState({ sortBy, sortDirection });
+    this.setState({ sortBy: header.id, sortDirection });
   };
 
   valueFromNode(item) {
     const mapping = {
-      'Instance Type': item.instanceType,
-      'Availability Zones': item.zone,
-      'Running Instances': item.running,
-      'Pending Instances': item.pending,
+      instanceType: item.instanceType,
+      availabilityZones: item.zone,
+      runningInstances: item.running,
+      pendingInstances: item.pending,
     };
```

Comment:
We should be able to do this without a `valueFromNode` method. We no longer need a mapping when we have the `id` key.

Reply:
After removing the `valueFromNode` method from ui/src/components/AwsProvisionerWorkerTypeStatus/index.jsx and ui/src/components/Ec2ResourcesTable/index.jsx, I discovered that the column contents no longer sort when the header is clicked. In ui/src/components/AwsProvisionerWorkerTypeStatus/index.jsx, I had to change the `id` keys in this way to enable sorting:
```
[
      { label: 'Instance Type', id: 'instanceType', type: 'string' },
      { label: 'Availability Zones', id: 'zone', type: 'string' },
      {
        label: 'Running Instances',
        id: 'running',
        type: 'number',
      },
      {
        label: 'Pending Instances',
        id: 'pending',
        type: 'number',
      },
    ];
```
They are no longer camel cased versions of the labels. Is this ok?

Reply:
That's fine :) You are doing it properly.